src/test/java/TriangleTest.java

```diff
@@ -0,0 +1,47 @@
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@Slf4j
+public class TriangleTest {
+
+    @BeforeAll
+    static void setupMsg() {
```

Comment:
takiej metody używamy raczej rzadko - już prędzej ma sens @BeforeEach bo jest w stanie przygotować dane przed każdym testem; ale akurat w tym przypadku to i to ma średni sens - nic tu specjalnie nie trzeba bowiem setupować

Reply:
słusznie, nie jest potrzebne - to tylko taki bonus żeby się pochwalić, że doczytałam :D

Reply:
słusznie, nie jest potrzebne - to tylko taki bonus żeby się pochwalić, że doczytałam :D